src/admin/components/ResourceTabs/PolygonReviewTab/components/PolygonStatus/StatusDisplay .tsx

```diff
@@ -0,0 +1,249 @@
+    </Text>
+  );
+
+  const openFormModalHandlerStatus = () => {
+    openModal(
+      <ModalConfirm
+        title={`${titleStatus} Status Change`}
+        commentArea
+        menuLabel={""}
+        menu={menuOptionsMap[titleStatus]}
+        onClose={closeModal}
+        content={contentStatus}
+        onConfirm={async (text: any, opt) => {
+          const option = menuOptionsMap[titleStatus].find(option => option.value === opt[0]);
+          let response;
+          try {
+            response = await mutate({
+              pathParams: { uuid: record?.uuid },
+              body: {
+                status: option?.status,
+                comment: text,
+                type: "status"
+              }
+            });
+            if (response.poly_id) {
+              setSelectedPolygon(response?.poly_id);
+            }
+            setNotificationStatus({
+              open: true,
+              message: "Your Status Update was just saved!",
+              type: "success",
+              title: "Success!"
+            });
+            setTimeout(() => {
+              setNotificationStatus({
+                open: false,
+                message: "",
+                type: "success",
+                title: "Success!"
+              });
+            }, 3000);
+          } catch (e) {
+            alert("The request encountered an issue, or the comment exceeds 255 characters.");
+            console.error(e);
+          } finally {
+            refresh();
+            reloadEntity();
+            closeModal;
+          }
+        }}
+      />
+    );
+  };
+
+  const openFormModalHandlerRequest = () => {
+    openModal(
+      <ModalConfirm
+        title={"Change Request"}
+        content={contentRequest}
+        commentArea
+        onClose={closeModal}
+        onConfirm={async (text: any, opt) => {
+          const option = menuOptionsMap[titleStatus].find(option => option.value === opt[0]);
+          try {
+            await mutate({
+              pathParams: { uuid: record?.uuid },
+              body: {
+                status: option?.status,
+                comment: text,
+                type: "change-request",
+                is_active: true,
+                request_removed: false
+              }
+            });
+            setNotificationStatus({
+              open: true,
+              message: "Your Change Request was just added!",
+              type: "success",
+              title: "Success!"
+            });
+            setTimeout(() => {
+              setNotificationStatus({
+                open: false,
+                message: "",
+                type: "success",
+                title: "Success!"
+              });
+            }, 3000);
+          } catch (e) {
+            alert("The request encountered an issue, or the comment exceeds 255 characters.");
+            console.error(e);
+          } finally {
+            refresh();
+            reloadEntity();
+            closeModal;
```

Comment:
`closeModal()`

Reply:
Thanks for pointing that out! I've updated the code to ensure closeModal is called as a function.